src/test/java/core/basesyntax/strategy/impl/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    @Test
+    public void supply88_Banana_Ok() {
```

Comment:
I think it's better not to use numbers in the names of test methods! Better focus on the result of the test!

Reply:
I think that this way our test will be more informative in the title